tests/manage/z_cluster/nodes/test_simultaneous_ocs_node_drains.py

```diff
@@ -0,0 +1,96 @@
+import logging
+
+import time
+
+from subprocess import TimeoutExpired
+
+from ocs_ci.ocs.cluster import validate_osd_pdb
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.node import drain_nodes, schedule_nodes, get_nodes
+from ocs_ci.framework.testlib import (
+    tier4b,
+    ManageTest,
+    ignore_leftovers,
+    skipif_lso,
+)
+from ocs_ci.ocs.resources.pod import wait_for_storage_pods
+
+
+log = logging.getLogger(__name__)
+
+
+@tier4b
+@skipif_lso
+@ignore_leftovers
+class TestSimultaneousOcsNodeDrains(ManageTest):
+    """
+    This class contains test for simultaneous node drains
+
+    """
+
+    def test_simultaneous_node_drains(
+        self,
+        pvc_factory,
+        pod_factory,
+        bucket_factory,
+        rgw_bucket_factory,
+        node_drain_teardown,
+    ):
+        """
+        - Check for OSD PDBs before drain
+        - Maintenance (mark as unschedulable and drain) 2 worker node with delay of 30 secs
+        - Drain will be completed on worker node A
+        - Drain will be pending on worker node B due to blocking PDBs
+        - Check the OSD PDBs
+        - Mark the node A as schedulable
+        - Let drain finish on Node B
+        - Mark the node B as schedulable
+        - Check cluster and Ceph health
+
+        """
+
+        # Validate OSD PDBs before drain operation
+        validate_osd_pdb()
+
+        # Get 2 worker nodes to drain
+        typed_nodes = get_nodes(node_type=constants.WORKER_MACHINE, num_of_nodes=2)
+        assert typed_nodes, "Failed to find worker nodes for the test"
```

Comment:
The condition is not fully covered if you get 1 node.
The assert will not trigger but you try to access to teped_node[1] and it will fail